src/app.tsx

```diff
@@ -17,6 +17,8 @@
  */
 
 import React, { FunctionComponent, ReactElement } from "react";
+import { ScriptBasedFlow } from "./components/script-based-flow";
```

Comment:
Let's export `./script-based-flow` from `components/index.ts`.
```suggestion
import { ScriptBasedFlow } from "./components";
```